doc/sphinx/proof-engine/ssreflect-proof-language.rst

```diff
@@ -203,7 +203,7 @@ with a function definition or Coq’s basic destructuring let. The let:
 construct differs from the latter in that
 
 
-+ The pattern can be nested (deep pattern matching), in particular,
++ The pattern can be nested (deep pattern matching); in particular,
```

Comment:
Yes, semicolon is sometimes used to separate clauses containing commas, but it's not common outside of legal text (e.g. laws passed by a legislative body) and the mostly doesn't use the semicolon this way.

Reply:
My understanding is that two sentences shouldn't be separated by anything but a period or a semicolon, unless there is a conjunction word in between.

Reply:
I'm sorry for the 3 PRs. In fact, I noticed the "Edit in Github" button on top of the documentation page, and then saw a message stating that PRs can be merged, so I tried to see how I could do that with another PR. But I couldn't find a way to do this on the Github web site. Do you know if this is possible, or do I have to clone therepo on my own computer to do that?

BWT, what would be a proper size for this type of documentation-related PR?

Reply:
Perhaps it would be best if these were 2 sentences then.

I think the file edit mechanism in github is rarely used.  In 4 years working on Coq I've never used it.  This is the first time I've seen someone else use it.  Most people clone the repo--this permits local testing (e.g. some changes to rst files will prevent the doc from building).  Merge refers to merging to master, which can only be done by maintainers; furthermore we use a different procedure to merge--not the built-in merge functionality.

You would have to clone the repo, then cherrypick 2 of the PRs onto the other and then force-push.  Are you familiar with doing such things in git?

Reply:
It was also a first for me to try this Edit button :) I was curious to see how it would work. But, since I couldn't manage to do what I wanted from the web site, I cloned the repo yesterday, and will thus make a PR once I'm done reading the documentation, which will be a good refresher anyway for the more subtle parts of SSReflect.

I'm not very familiar with Github, but I'll try to merge the patches together as you suggest.